latex/ledger-spec.tex

```diff
@@ -429,6 +435,94 @@ \section{Addresses}
 
 \clearpage
 
+\section{Protocol Parameters}
+\label{sec:protocol-parameters}
+
+The $\PParams$ is
+an abstract type that will represent an environment variable that contains
+values on which the functionality of the blockchain protocol depends, such
+as the fees transactions are obligated to pay to be processed.
+
+TODO - move explanations of indididual parameters explanation here.
+
+\begin{figure*}
+  \emph{Abstract types}
+  %
+  \begin{equation*}
+    \begin{array}{r@{~\in~}lr}
+      \var{fparams} & \type{FeeParams} & \text{min fee parameters}\\
+    \end{array}
+  \end{equation*}
+  %
+  \emph{Derived types}
+  %
+  \begin{equation*}
+    \begin{array}{r@{~\in~}l@{\qquad=\qquad}lr}
+      \var{coin}
+      & \Coin
+      & \mathbb{Z}
+      & \text{unit of value}
+      \\
+    \end{array}
+  \end{equation*}
+  %
+  \emph{Protocol Parameters}
+  %
+  \begin{equation*}
+    \PParams =
+    \left(
+      \begin{array}{r@{~\in~}lr}
+        \var{fparams} & \type{FeeParams} & \text{min fee parameters}\\
+        \var{keyDeposit} & \Coin & \text{stake key deposit}\\
+        \var{keyMinRefund} & \unitInterval & \text{stake key min refund}\\
+        \var{keyDecayRate} & \nonnegReals & \text{stake key decay rate}\\
+        \var{poolDeposit} & \Coin & \text{stake pool deposit}\\
+        \var{poolMinRefund} & \unitInterval & \text{stake pool min refund}\\
+        \var{poolDecayRate} & \nonnegReals & \text{stake pool decay rate}\\
+        \var{movingAvgWeight} & \unitInterval & \text{moving average weight}\\
+        \var{movingAvgExp} & \unitInterval & \text{moving average exponent}\\
+        \var{E_{max}} & \Epoch & \text{epoch bound on pool retirement}\\
+        \var{n_{opt}} & \mathbb{N} & \text{desired number of pools}\\
```

Comment:
I think it would make sense to explicitly make it an element of `N^+` as it is used in division later